source/genCases_bf16.c

```diff
@@ -350,6 +352,8 @@ static bfloat16_t bf16Random( void )
         return bf16RandomQInfP3();
      case 7:
         return bf16RandomQInfPInf();
+     default:
+        __builtin_unreachable();
```

Comment:
`__builtin_unreachable` is not standard C.  Please find a different way of expressing this that is standards-compliant.

Reply:
You might be able to use something like `(void) 0;`.

Reply:
Could we just use `abort()`? It's defined in `stdlib.h`

Reply:
Yeah, that is better than my suggestion.

Reply:
Updated.